### ISSUE TITLE
illegal instruction on Pentium II and earlier

### ISSUE THREAD

**Issue by asdfugil** [checkra1n/BugTracker#1734]
**Tell us about your setup:**
1. What iDevice are you using? iPad Pro 9.7
2. On what version of iOS is it?  13.5
3. What version of checkra1n are you using? 0.11.0
4. What is your host system (OS version? Hackintosh? VM? etc.)? Alpine linux on i486
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)? USB-A

**What are the steps to reproduce the issue?**
1. ./checkra1n
2.Illegal Instruction

**What do you expect, and what is happening instead?**
Since the checkra1n site says the build is "i486-pc", I expected it to work on a 486, however it crash on launch because of illegal instruction.
**Any other info, error logs, screenshots, ...?**
dmesg output:

```
[5672.233284] traps: checkra1n[4578] trap invalid opcode ip: 5b8e94 sp: bff8e700 error:0 in checkra1n[4a0000+bb3000]
```

**Comment by woachk** [comment]
Reproduced on my machine, now cooking a fix for that one. :)

edit: fixed, fix will be deployed in the next release.

**Comment by woachk** [comment]
https://assets.checkra.in/downloads/linux/cli/i486/fdcaaa64cf5f1034cbb729b5e421735189444da7cc66c9bd123db26a4b2da930/checkra1n

**Comment by asdfugil** [comment]
Just tested. This checkra1n build works and can detect my iPad on a 486.

**Comment by Siguza** [comment]
Marking this as fixed, and 0.12.0 should work out of the box now.
